data/HoldUpFjord.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "Dominick J Monaco",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Conservation Tech turned Software Engineer. Interested in contributing to community, climate change solutions, and dinner parties",
+  "avatar": "https://github.com/HoldUpFjord.png",
+   "tags": [
+    "100Devs",
+    "Green-tech",
+    "Javascript",
+    "Node.js",
+    "Tailwind",
+    "React",
+    "MongoDb",
+    ],
+  
+  "links": [
+    {
+      "name": "Follow my stream of consciousness",
+      "url": "https://twitter.com/JackMonaco1",
+      "icon": "FaTwitter"
+    }, {
+      "name": "Connect with me on Linkdln",
+      "url": "https://www.linkedin.com/in/dominick-j-monaco/",
+      "icon": "FaLinkdin"
+    }, {
+      "name": "Check out my github",
+      "url": "https://github.com/HoldUpFjord",
+      "icon": "FaGithub"
+    }, {
+      "name": "Here's my personal/portfolio site",
+      "url": "https://dominickjmonaco.netlify.app/",
+      "icon": "FaTwitter"
```

Comment:
I recommend updating it with any other icon. Because it will misguide your users with 2 twitter links! 
You can use link icon instead. 🛠

```suggestion
      "icon": "FaLink"
```

Reply:
Fixed both issues! Thank you for this one. Having two twitter icons was a sloppy pversight on my part.